src/templates/Autocomplete/.template.js

```diff
@@ -0,0 +1,14 @@
+const install = require('../../tasks/node/install');
+const teardown = require('../../tasks/node/teardown');
+
+module.exports = {
+  category: 'Web',
+  libraryName: '@algolia/autocomplete-js',
+  templateName: 'autocomplete',
+  appName: 'autocomplete-app',
+  keywords: ['algolia', 'Autocomplete', 'autocomplete'],
```

Comment:
Is it necessary to have two casing variations?

Reply:
Probably not—removed in c65154c99b967dcc44907b4e826c4b0c0c2aa8fd.